tests/models/bart/test_modeling_tf_bart.py

```diff
@@ -138,6 +150,23 @@ def check_decoder_model_past_large_inputs(self, config, inputs_dict):
         # test that outputs are equal for slice
         tf.debugging.assert_near(output_from_past_slice, output_from_no_past_slice, rtol=1e-3)
 
+    def create_and_check_bart_xla_generate_fast(self, config, input_ids, *args):
+        config.eos_token_id = None
```

Comment:
It was not clear to me why we want to set `eos_token_id` to `None`, until I searched in the library
```
# Generate until max length
``` 
It's probably my own issue (bad memory), but I would advocate to add comments.

Reply:
Added as a comment :D